librdmacm/rsocket.c

```diff
@@ -4678,8 +4678,11 @@ static void *cm_svc_run(void *arg)
 			fds[i].revents = 0;
 
 		poll(fds, svc->cnt + 1, -1);
-		if (fds[0].revents)
+		if (fds[0].revents) {
 			cm_svc_process_sock(svc);
+			/* svc->contexts will be changed, so need to assign again */
+			fds = svc->contexts;
```

Comment:
Please write an explanation to the change in commit message, together with exception.

We are following kernel coding style, so Fixes line should be added too.
Fixes: b60c79daaacc ("rsockets: Use service thread to accept connections")

@shefty

Reply:
nit: I would say 'svc->contexts may have been reallocated' rather than 'will be changed'  Fix looks good.